controls/datagrid/search-as-you-type.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Search as You Type
+page_title: .NET MAUI DataGrid Documentation - Search as You Type
+description: "Learn more about the search as you type feature of the Telerik .NET MAUI DataGrid - how to show/hide the search panel, various configuration settings of the search functionality, methods and events related to searching."
```

Comment:
* A well-thought-out description
* You may consider capitalizing the name of the feature as this will make it easier to identify.
```suggestion
description: "Learn more about the Search as You Type feature of the Telerik .NET MAUI DataGrid - how to show or hide the search panel, various configuration settings of the search functionality, methods, and events related to searching."
```